### DISCUSSION TITLE
App Widget Not Appearing on Category Widget Page 'Microsoft Store' - Need Help

### DISCUSSION THREAD

**Discussion by EdwinVD** [microsoft/WindowsAppSDK#3530] [Q&A] [answered]
Hi there, 
Could you please help me resolve the issue why my app widget is not appearing on the category widget page? I have already published the app on the Microsoft Store, but for some reason, the widget is not showing up where it should be. Can you please advise me on what steps I can take to fix this issue? Thank you.
<img width="1800" alt="Screen Shot 2023-03-23 at 4 54 08 PM" src="https://user-images.githubusercontent.com/16556638/227361790-61cc21ee-0f53-46f6-8e20-d4a922073ac9.png">

I would like to see the following apps, along with the widgets that I have recently created, to be visible on that page:
1. [My Lunar New Year](https://www.microsoft.com/store/apps/9P5Q01DM3S0S):  STORE: https://www.microsoft.com/store/apps/9P5Q01DM3S0S This widget helps users stay updated with the Lunar new year’s dates and events. Users can also learn more about the lunar next zodiac signs.
2. [My Christmas Tree](https://www.microsoft.com/store/apps/9WZDNCRDQGD7):  STORE: https://www.microsoft.com/store/apps/9WZDNCRDQGD7 As the name suggests, this widget brings the joy of the holiday season to your desktop. It allows users to decorate a virtual Christmas tree with various ornaments, lights, and animations.
3. [My Photo Slideshow](https://www.microsoft.com/store/apps/9PJJHLWNR5NN): STORE: https://www.microsoft.com/store/apps/9PJJHLWNR5NN  This widget shows you each chosen time a new photo, from your own photo folder or from the Bing Wallpaper.
4. [My Quick Launch](https://www.microsoft.com/store/apps/9NKGLX4G7DJX):  STORE: https://www.microsoft.com/store/apps/9NKGLX4G7DJX This widget is a convenient tool that allows users to access their most-used apps and programs with a single click.
5. [My Tic-Tac-Toe](https://www.microsoft.com/store/apps/9NGJDF77B98P):  STORE :https://www.microsoft.com/store/apps/9NGJDF77B98P A classic game, this widget lets users play the popular game of tic-tac-toe against the computer.
6. [Hard Disk](https://www.microsoft.com/store/apps/9NNCSN19QMCX): STORE: https://www.microsoft.com/store/apps/9NNCSN19QMCX  This widget displays the status of the user’s hard drive, providing information about the used and free space available. This widget can be useful for users who need to monitor their hard drive usage regularly.
7. [My Slider Puzzle](https://www.microsoft.com/store/apps/9P5ZS9SX49SP):  STORE: https://www.microsoft.com/store/apps/9P5ZS9SX49SP This widget is a game widget that allows you to create your own jigsaw puzzle using a custom image or Bing wallpaper. It is a fun and interactive way to pass the time and challenge your puzzle-solving skills.
8. [My Feed Reader](https://www.microsoft.com/store/apps/9NH53TM1KSK2): STORE: https://www.microsoft.com/store/apps/9NH53TM1KSK2 This widget allows users to keep up with their favorite websites and blogs by displaying the latest news and updates from their RSS feeds.
9. [Fast Shutdown](https://www.microsoft.com/store/apps/9WZDNCRDQGCS):  STORE: https://www.microsoft.com/store/apps/9WZDNCRDQGCS This widget provides a quick and easy way to shut down or restart the computer. This can be particularly useful for users who need to shut down their computer quickly without going through the usual shutdown process.

Please let me know what I need to do, and feel free to share any feedback with me. Thank you!

**Comment by rahulamlekar** [comment, marked as answer]
Hi Edwin, it's exciting to see these new widgets! They look great!

There's a simple form you can fill to request to be included in the Store collection page. Check out the publishing section of our [widgets dev docs](https://learn.microsoft.com/en-us/windows/apps/develop/widgets/implement-widget-provider-cs#publishing-your-widget).

**Comment by EdwinVD** [comment]
Thank you for the answer @rahulamlekar
